runbotics/runbotics-sdk/src/handler/StatefulActionHandler.ts

```diff
@@ -5,7 +5,7 @@ import { DesktopRunResponse } from "./DesktopRunResponse";
 export abstract class StatefulActionHandler {
     abstract run(
         request: DesktopRunRequest
-    ): Promise<DesktopRunResponse>;
+    ): Promise<DesktopRunResponse> | Promise<void>;
```

Comment:
imo you should change the type to `Promise<DesktopRunResponse | void>`, also can you give me a hint why we need to specify additionally `void` here?

Reply:
also remember to bump version before merge